draft-moriarty-attestationsets.md

```diff
@@ -19,12 +19,12 @@ venue:
   type: "Working Group"
   mail: "rats@ietf.org"
   arch: https://mailarchive.ietf.org/arch/browse/rats/
-  github: aj-stein-nist/draft-moriarty-attestationsets
-  latest: https://aj-stein-nist.github.io/draft-moriarty-attestationsets/draft-moriarty-attestationsets.html
+  github: kme/draft-moriarty-attestationsets
+  latest: https://kme.github.io/draft-moriarty-attestationsets/draft-moriarty-attestationsets.html
```

Comment:
One set of changes we needed to discuss and finalize is updating these links for the next revision you publish. It won't matter until then, but once it does we should have this changed to your repo-based website to be canonical version, not mine.